### ISSUE TITLE
ComboBox should react on PointerReleasedEvent instead of PointerPressedEvent

### ISSUE THREAD

**Issue by dmrayt** [AvaloniaUI/Avalonia#3428]
In Windows and also Linux the ComboBox react when the pointer is released not when it's pressed. Avalonia ComboBox should behave the same.

Now it's impossible to scroll with touch gestures in a ComboBox because the dropdown is closing as soon as is it touched.

**Comment by aguahombre** [comment]
Avalonia 0.9.4
I have found that I cannot select an item in a ComboBox dropdown using touch on a Raspberry Pi Linux app running in frame buffer mode. The same app runs fine under Linux X11 on the same hardware.

I can open the combo dropdown list normally using touch, but as soon as I touch an item in the list, the dropdown closes as though I had touched (moved focus to) another control outside the combobox.

Is this due to the bug described above?

Menus selections are working correctly using touch.
